app/src/main/java/com/vero/tasky/agenda/data/local/dao/EventDao.kt

```diff
@@ -0,0 +1,22 @@
+package com.vero.tasky.agenda.data.local.dao
+
+import androidx.room.*
+import com.vero.tasky.agenda.data.local.EventWithPhotosAndAttendees
+import com.vero.tasky.agenda.data.local.entities.EventEntity
+
+interface EventDao {
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    fun insertEvents(vararg events: EventEntity)
+
+    @Update
+    fun updateEvents(vararg events: EventEntity)
+
+    @Delete
+    fun deleteEvents(vararg events: EventEntity)
+
+    @Transaction
+    @Query("SELECT * FROM event")
+    fun loadEventsForDay(epoch: Long) : List<EventWithPhotosAndAttendees>
```

Comment:
Something I recommend is returning a Flow when using Room
Example:
```
fun loadEventsForDay(epoch: Long) : List<EventWithPhotosAndAttendees>
```

Same for Tasks and Reminders